index.html

```diff
@@ -94,8 +94,9 @@
       </pre>
 
       <ul>
+        <li>The <code><a>as</a></code> attribute is a required attribute.</li>
         <li>The specified <code><a>as</a></code> value MUST be a valid <a href="http://fetch.spec.whatwg.org/#concept-request-context">request context</a> as defined in [[FETCH]].</li>
-        <li>If the <code><a>as</a></code> attribute is missing, the user agent MUST assign a default context.</li>
+        <li>If the <code><a>as</a></code> attribute is omitted, or the value contains an invalid request context, then the user agent SHOULD output a developer-friendly warning and abort the fetch.</li>
```

Comment:
We can't enforce a MUST. E.g. there is no way to abort an inflight HTTP/1 request short of sending a TCP RST, which carries a high penalty and requirement to open a new connection... which may or may not make sense, and this should be deferred to the UA.